qemu/tests/driver_in_use.py

```diff
@@ -85,7 +85,14 @@ def run_subtest(sub_type):
 
         :params: sub_type: Sub test.
         """
-        utils_test.run_virt_sub_test(test, params, env, sub_type)
+        if params.get("bg_stress_test_is_cmd", "no") == "yes":
+            session = vm.wait_for_login()
+            sub_type = utils_misc.set_winutils_letter(
```

Comment:
Since this is a general function, is every referenced cmd(sub_type) neet to set winutils_letter ?

Reply:
Actually this is a specific scenario for rng test only, you can check driver_load_stress.cfg file.
run_bg_stress_test() function use this method for with_viorng.during_bg_test case.